src/test/client/ops.cc

```diff
@@ -0,0 +1,24 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2022 New Dream Network
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#include <iostream>
+#include <errno.h>
+#include "gtest/gtest.h"
+#include "TestClient.h"
+
+TEST_F(TestClient, CheckDummyOP) {
+  int res = client->check_dummy_op(myperm);
+  ASSERT_EQ(res, -EOPNOTSUPP);
+}
+
```

Comment:
Next, write tests for:

* reclaim with an invalid flag (-1?)
* reclaim with 0 flag
* session with invalid op (-1?) causes blocklisting of client

Reply:
There's only one flag at the moment for reclaim i.e. `CEPH_RECLAIM_RESET` having value `1`, reclaim with 0 as well -1 flag would fail with `EINVAL`. Should I just check with -1 or 0 in this case?

Reply:
only check the error conditions. And keep in mind older MDS return EOPNOTSUPP and not EINVAL.

Reply:
So the tests should pass either it encounters `EOPNOTSUPP || EINVAL`? Cause keeping only one of them would throw wrong results.

Reply:
added tests. PTAL